src/libcore/task/wake.rs

```diff
@@ -37,9 +36,9 @@ impl RawWaker {
     /// from a `RawWaker`. For each operation on the `Waker`, the associated
     /// function in the `vtable` of the underlying `RawWaker` will be called.
     #[rustc_promotable]
-    #[unstable(feature = "futures_api",
-            reason = "futures in libcore are unstable",
-            issue = "50547")]
+    #[cfg_attr(stage0, unstable(feature = "futures_api_const_fn_ptr", issue = "50547"))]
+    #[cfg_attr(not(stage0), stable(feature = "futures_api", since = "1.35.0"))]
+    #[cfg_attr(not(stage0), rustc_allow_const_fn_ptr)]
```

Comment:
Per https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/unstable.20fn.20ptr.20in.20const/near/162666717, please try and remove `rustc_allow_const_fn_ptr` from this since it shouldn't be necessary since no function pointer is accepted as an argument.